ui/js/components/elements/tree.jsx

```diff
@@ -38,34 +38,40 @@ d3.selection.prototype.tspans2 = function(lines, lh) {
 }
 
 
-d3.wordwrap = function (line, maxCharactersPerLine, maxLines = 2) {
-	var parts = line.split(/([^a-z][a-z]*)/), //split at non-words
+d3.wordwrap = function (line = '', maxCharactersPerLine = 43, maxLines = 3) {
+	const regexChecks = [/_/ig, /-/ig, /([A-Z])/g];
```

Comment:
many bot names are not following a standard naming structure... As such you can find the following names:
test_bot_123
test-bot-123
testBot123

This regex will check all forms and separate accordingly.